src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/process/DefaultProcessExitListener.java

```diff
@@ -19,8 +19,10 @@
 /**
  * Default process exit listener that simply captures the exit code and makes it available with a
  * getter. Before the process exit code is captured, it's initialized to -1.
+ *
+ * <p>When used with {@link com.google.cloud.tools.app.impl.cloudsdk.internal.sdk.CloudSdk.Builder},
+ * this allows checking of the success or failure of an action execution.
```

Comment:
I'm not really sure, but you save a few lines of code and the cost of the anonymous classes.